primitives/keystore/src/testing.rs

```diff
@@ -267,27 +286,57 @@ mod tests {
 		let secret_uri = "//Alice";
 		let key_pair = sr25519::Pair::from_string(secret_uri, None).expect("Generates key pair");
 
-		let transcript = sr25519::vrf::VrfTranscript::new(
+		let data = sr25519::vrf::VrfInput::new(
 			b"Test",
 			&[
 				(b"one", &1_u64.to_le_bytes()),
 				(b"two", &2_u64.to_le_bytes()),
 				(b"three", "test".as_bytes()),
 			],
-		);
+		)
+		.into_sign_data();
 
-		let result = store.sr25519_vrf_sign(SR25519, &key_pair.public(), &transcript);
+		let result = store.sr25519_vrf_sign(SR25519, &key_pair.public(), &data);
 		assert!(result.unwrap().is_none());
 
 		store
 			.insert(SR25519, secret_uri, key_pair.public().as_ref())
 			.expect("Inserts unknown key");
 
-		let result = store.sr25519_vrf_sign(SR25519, &key_pair.public(), &transcript);
+		let result = store.sr25519_vrf_sign(SR25519, &key_pair.public(), &data);
 
 		assert!(result.unwrap().is_some());
 	}
 
+	#[test]
+	fn vrf_output() {
+		let store = MemoryKeystore::new();
+
+		let secret_uri = "//Alice";
+		let pair = sr25519::Pair::from_string(secret_uri, None).expect("Generates key pair");
+
+		let input = sr25519::vrf::VrfInput::new(
+			b"Test",
+			&[
+				(b"one", &1_u64.to_le_bytes()),
+				(b"two", &2_u64.to_le_bytes()),
+				(b"three", "test".as_bytes()),
+			],
+		);
+
+		let result = store.sr25519_vrf_output(SR25519, &pair.public(), &input);
+		assert!(result.unwrap().is_none());
+
+		store
+			.insert(SR25519, secret_uri, pair.public().as_ref())
+			.expect("Inserts unknown key");
+
+		let preout = store.sr25519_vrf_output(SR25519, &pair.public(), &input).unwrap().unwrap();
+
+		let result = preout.make_bytes::<32>(b"rand", &input, &pair.public());
+		assert!(result.is_ok());
+	}
+
```

Comment:
It would be nice to check returning `None` scenario for non-existing mapping for `sr25519_vrf_output`. (The scnario from fn's doc).